http_test.go

```diff
@@ -67,12 +67,12 @@ func TestHTTPTimestampGranted(t *testing.T) {
 			Tag:       5,
 			FullBytes: []byte{5, 0},
 		},
-		NoNonce: true,
 	}
 	req, err := NewRequest(requestOpts)
 	if err != nil {
 		t.Fatalf("NewRequest() error = %v", err)
 	}
+	req.Nonce = nil
```

Comment:
Dont we need to remove this?

Reply:
This is a test case that the mocked server returns a pre-generated response that's without a nonce. 

We have other test cases that cover `nonce`.